Minor misc fixes

kouloumos opened the pull request:
These fixes are some issues that I found while trying to run the setup locally. You can find more information about each fix at the respective commit.

Extheoisah commented on api/.env.example:
```diff
@@ -1,15 +1,18 @@
-ALBY_SECRET_ID="secret"
+ALBY_CLIENT_ID="secret"
 ALBY_API_URL="https://api.alby.com"
+# We are using an LND node on voltage.cloud for staging and production
```
We shouldn't need to include comments in .env.sample

kouloumos replied:
You are right, I'll try to include those information in the README instead.

Extheoisah commented on api/.env.example:
```diff
@@ -1,15 +1,18 @@
 REDIS_HOST="localhost"
 REDIS_PORT=6379
 REDIS_PASSWORD="secret"
+# The GitHub webook on the repository is how we detect that a PR
```
same here. no comments
```suggestion

```

Extheoisah reviewed (changes requested):
looks good. Just a few nits

kouloumos commented:
Removed comments from `env` file per @Extheoisah suggestion.

Extheoisah commented:
> Removed comments from `env` file per @Extheoisah suggestion.

Will you be updating the README in this pr or in a later pr?

kouloumos commented:
> Will you be updating the README in this pr or in a later pr?

Later

Extheoisah commented:
Just realized you made the pr to main branch. Pls switch the base branch to dev

kouloumos commented:
> Just realized you made the pr to main branch. Pls switch the base branch to dev

Rebased on top of `dev`.
Also added another commit with a change in seeders for transcripts

Extheoisah reviewed (approved):
Thanks for fixing the seeders. That's q technical debt we needed to offload
